src/GitExtensions.Extensibility/build/GitExtensions.Extensibility.targets

```diff
@@ -28,10 +28,16 @@
     <!-- The prebuild event is available in VS projects referencing "GitExtensions.Extensibility". -->
     <!-- It downloads $(GitExtensionsReferenceVersion) from $(GitExtensionsReferenceSource) and extracts it to $(GitExtensionsPath) -->
     <Target Name="PreBuild" BeforeTargets="$(BuildDependsOn)">
-        <Message Text="Executing prebuild event provided by GitExtensions.Extensibility" />
+        <Message Text="Executing prebuild event provided by GitExtensions.Extensibility" Importance="high" />
+        <Message Text="[GitExtensions.Extensibility] Source: $(GitExtensionsReferenceSource)" Importance="high" />
+        <Message Text="[GitExtensions.Extensibility] Version: $(GitExtensionsReferenceVersion)" Importance="high" />
+        <Message Text="[GitExtensions.Extensibility] Download Path: $(GitExtensionsDownloadPath)" Importance="high" />
+        <Message Text="[GitExtensions.Extensibility] GitExtensions binaries path: $(GitExtensionsPath)" Importance="high" />
+        <Message Text="[GitExtensions.Extensibility Documentation] For overwrite and valid values, see: https://github.com/gitextensions/gitextensions.extensibility/blob/master/src/GitExtensions.Extensibility/build/GitExtensions.Extensibility.targets" Importance="high" />

```

Comment:
The plugin developer could overwrite the values of these properties to target a specific version like it is done here:
https://github.com/gitextensions/GitExtensions.GerritPlugin/blob/master/src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.csproj.user

For example to start migrating to a version not yet released (that's what I did to migrate the plugin I maintain).

The goal of this change is:
1. don't loose time trying to find against which version of GitExtensions we are building
2. easily know where is located the version downloaded (because from what I get, you have to delete the folder manually if you want to use another version)
3. add this line of documentation to make easier for plugin developer how to change/overwrite the values and which one are valid